src/amp-story-player/amp-story-player-impl.js

```diff
@@ -505,6 +506,7 @@ export class AmpStoryPlayer {
   updateCurrentIframe_(iframeIdx) {
     this.updateVisibilityState_(iframeIdx, VisibilityState.VISIBLE);
     this.updateIframePosition_(iframeIdx, IframePosition.CURRENT);
+    tryFocus(this.iframes_[iframeIdx]);
```

Comment:
Just to make sure: this code only triggers on Story navigation, and not on page load?

Reply:
Yep!